tests/test_skip_remote_data.py

```diff
@@ -3,9 +3,39 @@
 # by run_tests because it has the remote_data decorator.
 
 import pytest
-
-from astropy.utils.data import conf, download_file
-
+from contextlib import closing
+from six.moves.urllib.request import urlopen
+from tempfile import NamedTemporaryFile, gettempdir
+
+
+ASTROPY_DATA_URL = "http://data.astropy.org/"
+TIMEOUT = 10
+DL_BLOCK_SIZE = 2 ** 16
+
+
+def download_file(remote_url):
+    with closing(urlopen(remote_url, timeout=TIMEOUT)) as remote:
+        info = remote.info()
+        if 'Content-Length' in info:
+            try:
+                size = int(info['Content-Length'])
+            except ValueError:
+                size = None
+        else:
+            size = None
+
+        with NamedTemporaryFile(delete=False) as f:
+            try:
+                bytes_read = 0
+                block = remote.read(DL_BLOCK_SIZE)
+                while block:
+                    f.write(block)
+                    bytes_read += len(block)
+                    block = remote.read(DL_BLOCK_SIZE)
+            except BaseException:
+                if os.path.exists(f.name):
+                    os.remove(f.name)
+                raise
```

Comment:
Do we actually need something this complicated for the tests here? I think we just want to check if the URL is allowed/disallowed, so a simple ``urlopen/read`` might be ok here?

Reply:
Yes, you're right. I'll clean it up.